python/tests/res/enkf/test_res_config.py

```diff
@@ -316,10 +313,6 @@ def assert_ecl_config(self, ecl_config, config_data, working_dir):
                 ecl_config.getStartDate()
                 )
 
-        self.assertEqual(
-                config_data["ECLBASE"],
```

Comment:
Why did the ECLBASE check go?

Reply:
The `ECLBASE` and `JOBNAME` functionality was very similar, and has been unififed in #232 - obviously the test updates fell in thw wrong PR.

Reply:
I realized when I started looking at #232. Does this imply that we are also deprecating one of them in the interface and configuration file at some point? There is an `ECLBASE` in the config file loaded above and I would like it to be verified that it behaves as expected. If that means accessing `ecl base` or `jobname` is not important. Could we update this test either in this PR or #232?

Reply:
The intended behaviour is:

1. Both `JOBNAME` and `ECLBASE` can be used interchangeably in the config file. If both are present `JOBNAME`will be preferred.
2. If `ECLBASE` is present the `ecl_config_is_active( )` will return true.

I.e. the moral of the story is:

1. If you have eclipse simulations as part of your forward model - use `ECLBASE`.
2. If you do not have eclipse simulations as part of your forward model - use `JOBNAME`

I think this is tested as it should; but I guess I can look closer.

Reply:
Ah, I understand now! The `JOBNAME` is still tested and then everything is good.